pkg/cosign/tlog.go

```diff
@@ -137,20 +137,40 @@ func getTlogEntry(rekorClient *client.Rekor, uuid string) (*models.LogEntryAnon,
 	return nil, errors.New("empty response")
 }
 
+func proposedEntry(b64Sig string, payload, pubKey []byte) ([]models.ProposedEntry, error) {
+	var proposedEntry []models.ProposedEntry
+	signature, err := base64.StdEncoding.DecodeString(b64Sig)
+	if err != nil {
+		return nil, errors.Wrap(err, "decoding base64 signature")
+	}
+
+	if len(signature) == 0 {
```

Comment:
Thanks for tracking this down! I think I understand the logic, but would you mind leaving a comment here to explain it to future people?